_test/markdown-linter-style.rb

```diff
@@ -0,0 +1,11 @@
+all
```

Comment:
Changed the pathname to be _linter instead of _test in https://github.com/shredtechular/m-lab.github.io/commit/5ac4e73b2716cce1f79a34e5219c6dbac4d91e6c